src/app/cli/src/tests/coda_archive_node_test.ml

```diff
@@ -4,9 +4,40 @@ open Coda_base
 
 let name = "coda-archive-node-test"
 
+let runtime_config =
+  lazy
```

Comment:
since these test configurations are similar, could we have a default configuration, overriding individual fields when needed, instead of repeating the entire config for each test?

Reply:
Different sets of tests use different profiles, I'm not sure it would make sense to use this one as the default. For example, #5292 converts the bootstrap profile, #5440 converts the transactions profile

Reply:
It looks like the configs in this PR are the same, except for the ledger names. Could the commonality in a set of tests be factored out?

Reply:
I've moved the profiles into `Runtime_config.Test_configs`, for this and the other PRs in the chain.